### PR TITLE
[TE] Add support for configuring additional custom fields in Jira Alert

### PR DESCRIPTION
**Pull request by akshayrai**

### CODE REVIEW

File: thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/YamlResource.java [hunk 1]
```diff
@@ -1107,4 +1115,33 @@ public Response listYamls(
...
+    TaskContext taskContext = new TaskContext();
+    taskContext.setThirdEyeAnomalyConfiguration(config);
+
+    // Run the notification task. This will update the subscription watermark as well.
+    DetectionAlertTaskInfo taskInfo = new DetectionAlertTaskInfo(detectionAlertConfigId);
+    TaskRunner taskRunner = new DetectionAlertTaskRunner();
+    try {
+      taskRunner.execute(taskInfo, taskContext);
+    } catch (Exception e) {
+      LOG.error("Exception while triggering the notification task with id " + detectionAlertConfigId, e);
```
Comment (jihaozh) [inline thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/YamlResource.java:1143]: return the error message in the endpoint's response?

File: thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/YamlResource.java [hunk 2]
```diff
@@ -1107,4 +1115,33 @@ public Response listYamls(
...
+    }
+    LOG.info("Notification api triggered successfully");
```
Comment (jihaozh) [inline thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/YamlResource.java:1145]: return a success response here?

File: thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/YamlResource.java (hunk 1 above)
Comment (akshayrai) [reply thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/YamlResource.java]: Done

File: thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/YamlResource.java (hunk 2 above)
Comment (akshayrai) [reply thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/YamlResource.java]: done

**Pull request merged**